composer.json

```diff
@@ -45,7 +45,8 @@
         "magento/module-inventory-sales": ">=1.1.0",
         "magento/module-inventory-indexer": ">=2.0.0",
         "magento/magento-composer-installer": "*",
-        "elasticsearch/elasticsearch": "~6.7|~7.2"
+        "elasticsearch/elasticsearch": "~6.7|~7.2",
+        "smile/magento2-smilelab-phpcs": "^2.0"
```

Comment:
This should not be in "require" part of the composer.json but remains in require-dev.